.circleci/config.yml

```diff
@@ -306,4 +307,3 @@ workflows:
           filters:
             branches:
               only: master
-
```

Comment:
Is this a manual edit, or a prettier edit-on-save?

Reply:
It wasn't an intentional manual edit...